pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -57,6 +58,15 @@ func (computeCfg *computeServiceConfig) getNetworkInterfacesOfVnet(vnetIDSet map
 	return nwIntfs, err
 }
 
+func (computeCfg *computeServiceConfig) getCacheNetworkInterfacesOfVnet(vnet string) []*networkInterfaceTable {
+	cachedNetworkInterfaces := computeCfg.getManagedNetworkInterfaces()
+	var networkInterfaces []*networkInterfaceTable
+	if nIfs, ok := cachedNetworkInterfaces[vnet]; ok {
```

Comment:
Don't we have to convert vnet to lowercase here ? When we add to the network interface map(snapshot) , we convert to lower case L262.